api/src/main/java/org/eclipse/microprofile/lra/annotation/Compensate.java

```diff
@@ -127,7 +127,7 @@
  *   <td scope="row">The resource does not know about the LRA</td>
  * </tr>
  * <tr>
- *   <td scope="row">500</td>
```

Comment:
I have two points that are not major ones.
* It could be nicer if the table content would be ordered by error code. That mean the line of table with cod `409` belongs before the line with code `410`.
* As the text below says it's possible that the implementation returns the `500`. Is it obvious by the HTTP behaviour or should that code be part of the table?

Reply:
> As the text below says it's possible that the implementation returns the 500. Is it obvious by the HTTP behaviour or should that code be part of the table?

HTTP behavior, this is not a supported code returned by the implementation of LRA but the general error which can be returned from JAX-RS server (same as connection error for instance) so it shouldn't be mentioned in the table.